### DISCUSSION TITLE
Can't hide sessions name

### DISCUSSION THREAD

**Discussion by morenohernan** [zellij-org/zellij#2897] [Q&A] [answered]
Hi, team.
I want to hide the session name. Reading the docs I've found this: 
<img width="1095" alt="image" src="https://github.com/zellij-org/zellij/assets/700619/9c06741e-5d2a-4a95-aa72-51db59160b97">

What is the "ui" element? I've tried to use it within a custom layout but I get errors.

**Comment by morenohernan** [comment]
More info.. 
Within my custom layout...
layout {
    tab_template name="ui" {
       pane size=1 borderless=true {
           plugin location="zellij:tab-bar"
       }
       children
       pane size=1 borderless=true {
           plugin location="zellij:status-bar"
       }
    }
......
.....
    ui name="Terminal"{
        pane
    }
.....
} // end layout

**Reply by Lockszmith-GH** [reply]
`ui` is the template defined at the top.

Have you tried to add the hide... settings to the template?

**Comment by morenohernan** [comment]
Yeap, I've tried as below:

    tab_template name="ui" {
       hide_session_name true
       pane size=1 borderless=true {
           plugin location="zellij:tab-bar"
       }
       children
       pane size=1 borderless=true {
           plugin location="zellij:status-bar"
       }
    }

Also like this:

    tab_template name="ui" {
        pane_frames {
            hide_session_name true
        }       
       pane size=1 borderless=true {
           plugin location="zellij:tab-bar"
       }
       children
       pane size=1 borderless=true {
           plugin location="zellij:status-bar"
       }
    }

But I get errors with those settings :
<img width="1399" alt="image" src="https://github.com/zellij-org/zellij/assets/700619/fc020b77-9a17-4002-ba93-73b2a956ce37">

**Reply by Lockszmith-GH** [reply, marked as answer]
Note that when you reply, you can reply to a thread, or open a new thread.

ok, I took another look at the documentation.

Your example shows a pane template named  `ui`, but that's not what the documentation is showing. 
The [hide_session_name](https://zellij.dev/documentation/options.html?highlight=hide_session_name#hide_session_name) is a `ui` setting in the [`config.kdl` file](https://zellij.dev/documentation/options.html?highlight=hide_session_name#options), not a layout.
